### PR TITLE
Document default Electron offline support

### LINKED ISSUE #5364: [Electron] Offline Integration
### Core or SDK?

Platform/SDK

### Which part? Which one?

Electron

### Description

A customer is requesting we provide an example of how to use the Offline integration for the Electron SDK. 

### Suggested Solution

Just adding a simple message or so, on how we they can use it for their SDK if needed or not.

### PR DESCRIPTION
**Pull request by timfish**
Closes #5364

### CODE REVIEW

File: src/platforms/javascript/guides/electron/index.mdx [hunk 1]
```diff
@@ -82,6 +82,33 @@ init({ /* config */ }, reactInit);
 
 ```
 
+## Offline Support
+
+The default transport automatically handles offline caching of events. It has a
+number of options that allow customizing queueing behavior:
```
Comment (lizokm) [inline src/platforms/javascript/guides/electron/index.mdx:88]:
```suggestion
The default transport automatically handles offline events caching. Following, are a
number of options that allow you to customize queueing behavior:
```

**Review by lizokm — COMMENTED** [review]
Made a few small tweaks.

**Review by AbhiPrasad — APPROVED** [review]

**Comment by timfish** [comment]
A couple of the properties added in this documentation are not out in a release yet but I'll mark this as ready when they are!

**Comment by timfish** [comment]
This can now be merged since `@sentry/electron` v4.1.1 has been released!

**Comment by lforst** [comment]
Thank you Tim!

**Pull request merged**